packages/trpc/server/routers/loggedInViewer/updateProfile.handler.ts

```diff
@@ -320,7 +316,7 @@ export const updateProfileHandler = async ({ ctx, input }: UpdateProfileOptions)
     });
   }
 
-  if (updatedUser && hasEmailChangedOnCalProvider) {
+  if (updatedUser) {
```

Comment:
Woops, not putting hasEmailBeenChanged here caused email verification emails to be sent on every user profile update 👎

Reply:
Oversight from my side, sorry about that. Thanks for fixing it :)